integration-tests/functions/__tests__/__snapshots__/functions-dev.js.snap

```diff
@@ -1,5 +1,25 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`develop dynamic routes named wildcard routes 1`] = `
+Object {
+  "foo": "super",
+}
+`;
+
+exports[`develop dynamic routes param routes 1`] = `
+Object {
+  "super": "additional",
+  "userId": "23",
+}
+`;
+
+exports[`develop dynamic routes unnamed wildcard routes 1`] = `
+Object {
+  "*": "super",
+  "0": "super",
```

Comment:
send the path-to-regexp param style route so we avoid a breaking change until v5.